app/styles/components/_footer.scss

```diff
@@ -59,12 +82,25 @@ $icon-size: 25px;
 @media (max-width: $Theme-break-mobile) {
   .Footer {
     padding: $Theme-margin-half $Theme-margin-small;
+    display: flex;
+    flex-direction: column;
+  }
+
+
+  .Footer-column {
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
Assuming you want to center the column, you are already using flexbox which has the option to center elements.